src/fizzBuzz.js

```diff
@@ -0,0 +1,20 @@
+export default function fizzBuzz() {
+  var numbers = []
+  for (var number = 1; number <= 100; number++) {
```

Comment:
most people utilize the variable i for loop iteration.  We want to be as clear as possible with our variable names an number and numbers are too similar.